bindings/java/src/org/sleuthkit/datamodel/TaggingManager.java

```diff
@@ -140,9 +140,10 @@ public TagSet addTagSet(String name, List<TagName> tagNames) throws TskCoreExcep
 	}
 
 	/**
-	 * Remove a row from the tag set table. The TagNames in the TagSet will not
-	 * be deleted, nor will any tags with the TagNames from the deleted tag set
-	 * be deleted.
+	 * Remove a row from the tag set table. If the given TagSet has a valid list
+	 * of TagNames the TagNames will be removed from the tag_name table if there
+	 * are not references to the TagNames in the content_tag or
+	 * blackboard_artifact_tag table.
 	 *
 	 * @param tagSet TagSet to be deleted
```

Comment:
I know this wasn't this series of PR's, but I think this parameter description needs a period at the end.